<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugCreateStringValuesDialog</class>
 <widget class="QDialog" name="DebugCreateStringValuesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="apLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address Page:   0x</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>16383</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="apEdit">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>18</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hh</string>
   </property>
  </widget>
  <widget class="QLabel" name="paramLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parameter:    0x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="paramEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>18</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hh</string>
   </property>
  </widget>
  <widget class="QLabel" name="rangeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Range:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lowerRangeEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>58</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="upperRangeEdit">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>58</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="raw2valFileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Raw 2 Val File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>184</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="raw2valFileEdit">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>108</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="raw2ValFileButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>107</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="val2rawFileEdit">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>138</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="val2rawFileButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>137</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="val2rawFileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Val 2 Raw File:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
